qa/vagrant/src/main/java/org/elasticsearch/packaging/test/ArchiveTestCase.java

```diff
@@ -69,25 +72,42 @@
 @TestCaseOrdering(TestCaseOrdering.AlphabeticOrder.class)
 public abstract class ArchiveTestCase extends PackagingTestCase {
 
-    private static Installation installation;
+    public void test10Install() {
+        installation = installArchive(distribution());
+        verifyArchiveInstallation(installation, distribution());
+    }
 
-    /** The {@link Distribution} that should be tested in this case */
-    protected abstract Distribution distribution();
+    public void test11DebDependencies() {
+        assumeTrue(Platforms.isDPKG());
 
-    @BeforeClass
-    public static void cleanup() {
-        installation = null;
-        cleanEverything();
-    }
+        final Shell sh = new Shell();
 
-    @Before
-    public void onlyCompatibleDistributions() {
-        assumeTrue("only compatible distributions", distribution().packaging.compatible);
+        final Result defaultResult = sh.run("dpkg -I " + getDistributionFile(DEFAULT_DEB));
+        final Result ossResult = sh.run("dpkg -I " + getDistributionFile(OSS_DEB));
+
+        TestCase.assertTrue(Pattern.compile("(?m)^ Depends:.*bash.*").matcher(defaultResult.stdout).find());
+        TestCase.assertTrue(Pattern.compile("(?m)^ Depends:.*bash.*").matcher(ossResult.stdout).find());
+
+        TestCase.assertTrue(Pattern.compile("(?m)^ Conflicts: elasticsearch-oss$").matcher(defaultResult.stdout).find());
+        TestCase.assertTrue(Pattern.compile("(?m)^ Conflicts: elasticsearch$").matcher(ossResult.stdout).find());
     }
 
-    public void test10Install() {
-        installation = installArchive(distribution());
-        verifyArchiveInstallation(installation, distribution());
+    public void test12RpmDependencies() {
+        assumeTrue(Platforms.isRPM());
+
+        final Shell sh = new Shell();
+
+        final Result defaultDeps = sh.run("rpm -qpR " + getDistributionFile(DEFAULT_RPM));
+        final Result ossDeps = sh.run("rpm -qpR " + getDistributionFile(OSS_RPM));
+
+        TestCase.assertTrue(Pattern.compile("(?m)^/bin/bash\\s*$").matcher(defaultDeps.stdout).find());
+        TestCase.assertTrue(Pattern.compile("(?m)^/bin/bash\\s*$").matcher(ossDeps.stdout).find());
+
+        final Result defaultConflicts = sh.run("rpm -qp --conflicts " + getDistributionFile(DEFAULT_RPM));
+        final Result ossConflicts = sh.run("rpm -qp --conflicts " + getDistributionFile(OSS_RPM));
+
+        TestCase.assertTrue(Pattern.compile("(?m)^elasticsearch-oss\\s*$").matcher(defaultConflicts.stdout).find());
+        TestCase.assertTrue(Pattern.compile("(?m)^elasticsearch\\s*$").matcher(ossConflicts.stdout).find());
```

Comment:
If we're relocating these tests into a `PackagingTestCase` subclass would it make more sense for them to go in `PackageTestCase` (I now see the issue with these names) rather than `ArchiveTestCase`

Reply:
I don't think they make sense at the PackagingTestCase level, since they are specific to rpm/deb. I moved them into the oss rpm/deb tests, since they only need to run once.